rllib/core/learner/learner.py

```diff
@@ -1382,7 +1382,10 @@ def _update_from_batch_or_episodes(
         # Convert input batch into a tensor batch (MultiAgentBatch) on the correct
         # device (e.g. GPU). We move the batch already here to avoid having to move
         # every single minibatch that is created in the `batch_iter` below.
-        if self._learner_connector is None:
+        # Note, if we have a learner connector, but a `MultiAgentBatch` is passed in,
+        # we are in an offline setting.
+        # TODO (simon, sven): Check, if DreamerV3 has the same setting.
```

Comment:
We'll see what tests doing :) DreamerV3 does not use connectors. It passes the batch from the replay buffer directly into `update_from_batch`.